lib/mliropencl/include/pocl/Dialect/Dialect.h

```diff
@@ -0,0 +1,12 @@
+#ifndef POCL_DIALECT_H
+#define POCL_DIALECT_H
+
+#include "mlir/IR/Dialect.h"
```

Comment:
<>

Reply:
Went through these with grep and I should've now fixed every one except the .td files.